scripts/upgrade

```diff
@@ -28,11 +29,12 @@ db_name=$(ynh_app_setting_get $app db_name)
 # CHECK VERSION
 #=================================================
 
-ynh_abort_if_up_to_date
+upgrade_type=$(ynh_check_app_version_changed)
```

Comment:
It does not matter, but why do you store this and you never use it?

Reply:
In this very app, we don't use that variable because the upgrade script doesn't upgrade the upstream app.
Otherwise, we used that variable to know if it's only an upgrade of the package, or an upgrade of the upstream app as well.

So, that right that we could get rid of that variable in this package. But I preferred keeping it in case we would have anything to add regarding the upstream app in this upgrade script.